docs/storage_v2/built_in_features/drive/lifecycle.md

```diff
@@ -0,0 +1,86 @@
+---
+id: lifecycle
+title: Drive Lifecycle
+sidebar_label: Lifecycle
+---
+>**Note**\
+You can see the [bcdocs](https://bcdocs.xpxsirius.io/docs/built-in-features/drive) for getting information about schemas of transactions described here.
+
+## Create the drive
+
+To create the [Drive](overview.md), the [owner](../../roles/owner.md) must define Drive Size and Replicators Count and send Prepare Drive transaction. Drive Size should not be less than 1MB. Drive Size should be the integer number of MB. Replicator Number should be the integer number not less than 4. After [Drive](overview.md) creation, it should transfer the sufficient amount [XPX](../../getting_started/economy.md#xpx) to the drive’s balance. These mosaics will be exchanged to [SO](../../getting_started/economy.md#so) by [Replicator Nodes](../../roles/replicator.md) according to the drive size. The [Drive](overview.md) preparation has no strict time bounds and you will wait replicators assignment to the drive. Any [Replicator Node](../../roles/replicator.md) can join it at any time.
```

Comment:
I think it could be one sentence

`Drive Size should not be less than 1MB. Drive Size should be the integer number of MB.`

`
Replicator Number should be the integer number not less than 4
`

Min count of replicators defined in the Blockchain config. By default it 4, but it could change.